egcg_core/clarity.py

```diff
@@ -87,6 +87,16 @@ def get_species_from_sample(sample_name):
             return get_species_name(species_string)
 
 
+def get_genome_version(dataset_name, species):
+    genome_version = get_sample(dataset_name).udf.get('Genome Version')
+    if genome_version is None:
+        genome_version = cfg.query('species', species, 'default')
+    reference = cfg.query('genomes', genome_version, 'fasta')
```

Comment:
I would return genome_version here.
No need to return the fasta file as it is outside the scope of the function